pom.xml

```diff
@@ -56,7 +56,9 @@
       <module>rhq-plugin</module>
       <module>osgi</module>
       <module>spring/spring</module>
-      <module>spring/spring4</module>
```

Comment:
Unfortunately, if I then create an application which depends on spring4-embedded it won't find the parent pom in the repository